### PR TITLE
[MRG + 1] Clean up config, catch up w attelo PR #33 and #35

### PR DESCRIPTION
**Pull request by kowey**

### CODE REVIEW

File: irit_rst_dt/config/common.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    """return a key from a list of objects that have a
+    `key` field each"""
+    return '-'.join(v if isinstance(v, six.string_types) else v.key
+                    for v in variants)
+
+
+Settings = namedtuple('Settings',
+                      ['key', 'intra', 'oracle', 'children'])
+"""
+Note that this is subclass of Keyed
```
Comment (moreymat) [inline irit_rst_dt/config/common.py]: I am not sure I see how this is a subclass of Keyed.

Comment (kowey) [inline irit_rst_dt/config/common.py]: It has a key. Not a subclass, sorry, brain fart. Corrected doc string
should talk about just the fact that it has a key and can thus be used for
combined keys. Type error in my docs, sheesh.

File: irit_rst_dt/config/common.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+ORACLE = LearnerConfig(attach=attach_learner_oracle(),
+                       label=label_learner_oracle())
+
+# ---------------------------------------------------------------------
+# baselines
+# ---------------------------------------------------------------------
+
+
+def decoder_last():
+    "our instantiation of the mst decoder"
```
Comment (moreymat) [inline irit_rst_dt/config/common.py]: Could you fix the docstring for both baseline decoders?

File: irit_rst_dt/local.py [hunk 3]
```diff
@@ -252,242 +185,171 @@ def label_learner_dp_pa():
...
+EVALUATIONS = _evaluations()
 
 
 GRAPH_DOCS = [
-    'wsj_1184.out',
-    'wsj_1120.out',
-    ]
+    's2-league4-game1_07_stac_1396964826',
+    's2-league4-game1_02_stac_1396964918',
+]
```
Comment (moreymat) [inline irit_rst_dt/local.py]: Note these are STAC dialogues in irit-rst-dt :-)

**Comment by moreymat** [comment]
Apart from the above, LGTM.

**Pull request merged**